pkg/converter/convert_unix.go

```diff
@@ -684,13 +690,17 @@ func mergeLayers(ctx context.Context, cs content.Store, descs []ocispec.Descript
 		return nil, errors.Wrap(err, "marshal blob ids")
 	}
 
+	if opt.FsVersion == "" {
+		opt.FsVersion = "5"
```

Comment:
Should we move to fs version 6 as default?  Fs version 5 is really a burden for development so far.

Reply:
We have left `v5` as the default option in all components (nydusify/buildkit/nerdctl/acceld) before, maybe we can push `v6` as default together in the future.